src/Http/Routing/test/testassets/RoutingSandbox/HelloExtension/EndpointRouteBuilderExtensions.cs

```diff
@@ -13,19 +13,19 @@ namespace Microsoft.AspNetCore.Builder
 {
     public static class EndpointRouteBuilderExtensions
     {
-        public static IEndpointConventionBuilder MapHello(this IEndpointRouteBuilder builder, string template, string greeter)
+        public static IEndpointConventionBuilder MapHello(this IEndpointRouteBuilder routes, string pattern, string greeter)
         {
-            if (builder == null)
+            if (routes == null)
             {
-                throw new ArgumentNullException(nameof(builder));
+                throw new ArgumentNullException(nameof(routes));
             }
 
-            var pipeline = builder.CreateApplicationBuilder()
+            var pipeline = routes.CreateApplicationBuilder()
                .UseHello(greeter)
                .Build();
 
-            return builder.Map(
-                template,
+            return routes.Map(
+                pattern,
                 "Hello " + greeter,
```

Comment:
Is this the display name? What extension method is this?

Reply:
Not sure why I didn't hit an issue with this building locally. Fixed.